docs/api/algorand/transfer-tokens.mdx

```diff
@@ -0,0 +1,109 @@
+# TransferTokens
+
+Send DeCash from a wallet to another.
```

Comment:
*one wallet to another. This endpoint work for both regular and delegated wallets.